quaint/src/visitor/postgres.rs

```diff
@@ -499,6 +500,45 @@ impl<'a> Visitor<'a> for Postgres<'a> {
         }
     }
 
+    #[cfg(any(feature = "postgresql", feature = "mysql"))]
+    fn visit_json_array_agg(&mut self, array_agg: JsonArrayAgg<'a>) -> visitor::Result {
+        self.write("JSONB_AGG")?;
+        self.surround_with("(", ")", |s| s.visit_expression(*array_agg.expr))?;
+
+        Ok(())
+    }
+
+    #[cfg(any(feature = "postgresql", feature = "mysql"))]
+    fn visit_json_build_object(&mut self, build_obj: JsonBuildObject<'a>) -> visitor::Result {
+        const MAX_FIELDS: usize = 50;
```

Comment:
Would be great if you could add a comment that explains _why_ we're chunking in the first place. Linking the issue it fixes would also be useful